defs.bzl

```diff
@@ -46,9 +47,6 @@ pip_repository = repository_rule(
         "python_interpreter": attr.string(default="python3"),
         # 600 is documented as default here: https://docs.bazel.build/versions/master/skylark/lib/repository_ctx.html#execute
         "timeout": attr.int(default = 600),
-        "_script": attr.label(
```

Comment:
This was originally used to find the location of the rules. This logic has since moved. The refactoring now supports running the directory as a module.

This means we can call the function like so:
```
python3 -m extract_wheels -h
```
Without any mangling of the PYTHONPATH, or sys.path hacks